src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = people.filter(person => person.sex === 'm');
+  const result = !century ? man
+    : man.filter(person => Math.ceil(person.died / 100) === century);
+
+  return result
+    .map(person => person.died - person.born)
+    .reduce((sum, life) => sum + life, 0) / result.length;
```

Comment:
move it to the separate fucntion that you will use here and in the next function